### DISCUSSION TITLE
Rebuild the kernel

### DISCUSSION THREAD

**Discussion by vinhjaxt** [devmfc/debian-on-amlogic#11] [General]
I noticed that your kernel is missing some options.
```
CONFIG_COMPAT is not set
```
So I want to build a new one.
How do I do it?

Thank you very much!

**Comment by devmfc** [comment]
Yes, it is disabled since the 6.1.47 release, because it is advised to disable it by some security hardening sources. It is only needed for 32-bit user-space programs.  I'm curious what problem you encounter? If it is something that is used a lot then I might re-enable it in next builds.

To build a kernel, just Google it. The image has the used kernel config files in the /boot directory.
